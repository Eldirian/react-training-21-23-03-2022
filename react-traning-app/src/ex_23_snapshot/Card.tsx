import { getApplicationColorsScheme } from "./utils"

const Card = () => {
    return <div style={{
        backgroundColor: getApplicationColorsScheme().primary
    }}>
        Lorem ipsum
    </div>
}

export { Card }