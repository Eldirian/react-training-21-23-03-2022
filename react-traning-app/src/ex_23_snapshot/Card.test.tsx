import renderer from 'react-test-renderer';
import { Card } from './Card';


describe('Card', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Card />);
        expect(tree).toMatchSnapshot();
    })

})