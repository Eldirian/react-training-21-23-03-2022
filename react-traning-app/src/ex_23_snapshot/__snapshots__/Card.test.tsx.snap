// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "green",
    }
  }
>
  Lorem ipsum
</div>
`;
